docs/content/en/docs/Tutorials/phoenix-tutorial.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Use Phoenix with Reva"
+linkTitle: "Use Phoenix with Reva"
+weight: 5
+description: >
+  Connect Phoenix and Reva locally
+---
+
+This is a guide on how you can run both Pheonix and Reva locally in a dev environment. Phoenix is a frontend application connected to an open cloud backend server, e.g. Reva.
```

Comment:
Instead of "e.g. Reva.", you can say that it connects to the backend server through the Reva platform.